Helper/Search/QueryBuilder.php

```diff
@@ -157,7 +158,7 @@ private function getAggs($hasPostFilter = false): ?array
 
     private function getAgg(Filter $filter): ?array
     {
-        $agg = ['terms' => ['field' => $filter->getField(), 'size' => $filter->getAggSize()]];
+        $agg = ['terms' => ['field' => $filter->getField(), 'size' => $filter->getAggSize()], 'aggs' => [ 'reversed_nested' => [ 'reverse_nested' => new stdClass() ]]];
```

Comment:
I think this part should be part of the Filter class and not hard coded. We only want to activate this 'reversed_nested' agg in some cases. So Filter would be used to configure this behaviour ?